docker/Dockerfile

```diff
@@ -63,23 +63,25 @@ RUN /opt/pushd-dither/vendor/provision-vendored.sh
 
 # ====== END PUSHD DITHER ========
 
-
+COPY --from=build /opt/imgproxy/ /opt/imgproxy/
+COPY --from=build /usr/local/bin/imgproxy /usr/local/bin/imgproxy
 COPY docker/entrypoint.sh /usr/local/bin/
-COPY --from=build /usr/local/bin/imgproxy /usr/local/bin/
-COPY --from=build /usr/local/lib /usr/local/lib
 
 COPY NOTICE /usr/local/share/doc/imgproxy/
 
 ENV VIPS_WARNING=0
 ENV MALLOC_ARENA_MAX=2
-ENV LD_LIBRARY_PATH /usr/local/lib
```

Comment:
sure you don't need LD_LIBRARY_PATH=/opt/imgproxy/ ?

Reply:
this line is gone from the upstream, and seems to have no trouble finding its lib dir